saleor/settings.py

```diff
@@ -719,4 +723,6 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
     seconds=parse(os.environ.get("JWT_TTL_REQUEST_EMAIL_CHANGE", "1 hour")),
 )
 
-# Support multiple interface notation in schema for Apollo tooling.
+UPDATE_SEARCH_VECTOR_INDEX_QUEUE_NAME = os.environ.get(
+    "UPDATE_SEARCH_VECTOR_INDEX_QUEUE_NAME", "update-products-search-vector"
+)
```

Comment:
I would say to put it to `None` by default, otherwise users will need to setup a separate Celery worker by default when they may not actually need it

Reply:
Done